Adding Link to Environment Page for "Actions / Permission Statement Secrets Environments" (#24909)
github/docs: data/reusables/actions/permissions-statement-secrets-environment.md

> To create secrets {% ifversion actions-configuration-variables %}or variables {% endif %}for an environment in a personal account repository, you must be the repository owner. To create secrets {% ifversion actions-configuration-variables %}or variables {% endif %}for an environment in an organization repository, you must have `admin` access. Please visit "[AUTOTITLE](/actions/deployment/targeting-different-environments/using-environments-for-deployment)" to see if environments are available for your repository.

Reviewer: ```suggestion
To create secrets {% ifversion actions-configuration-variables %}or variables {% endif %}for an environment in a personal account repository, you must be the repository owner. To create secrets {% ifversion actions-configuration-variables %}or variables {% endif %}for an environment in an organization repository, you must have `admin` access. For more information on environments, see "[AUTOTITLE](/actions/deployment/targeting-different-environments/using-environments-for-deployment)."
```
I'm just updating the phrasing slightly to match our style guide around links. 💛
